support/cas-server-support-saml/src/main/java/org/apereo/cas/support/saml/util/AbstractSaml20ObjectBuilder.java

```diff
@@ -442,8 +442,8 @@ public String decodeSamlAuthnRequest(final String encodedRequestXmlString) {
         final String inflated = CompressionUtils.inflate(decodedBytes);
         if (!StringUtils.isEmpty(inflated)) {
             return inflated;
+        } else {
```

Comment:
This is fine, but the else clause is redundant here.

Reply:
A matter of style I would say, but yes, let's remove the `else` clause...